Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/API/UI/Call.hs

```diff
@@ -102,8 +104,8 @@ callStatusCallback = withFlowHandlerAPI . DCall.callStatusCallback
 getCallStatus :: Id CallStatus -> (Id Person.Person, Id DM.Merchant) -> FlowHandler DCall.GetCallStatusRes
 getCallStatus callStatusId _ = withFlowHandlerAPI $ DCall.getCallStatus callStatusId
 
-directCallStatusCallback :: Text -> ExotelCallStatus -> Maybe Text -> Maybe Int -> Maybe Int -> FlowHandler DCall.CallCallbackRes
-directCallStatusCallback callSid dialCallStatus_ recordingUrl_ duration = withFlowHandlerAPI . DCall.directCallStatusCallback callSid dialCallStatus_ recordingUrl_ duration
+directCallStatusCallback :: Text -> ExotelCallStatus -> Maybe Text -> Maybe Int -> Maybe Int -> Maybe Text -> Maybe Text -> FlowHandler DCall.CallCallbackRes
+directCallStatusCallback callSid dialCallStatus_ recordingUrl_ duration duration' _ _ = withFlowHandlerAPI $ DCall.directCallStatusCallback callSid dialCallStatus_ recordingUrl_ duration duration'
```

Comment:
what is the use of these new parameters?

Reply:
Knowlarity will be sending separate status for LegA and LegB in a call along with DialCallStatus as https://github.com/nammayatri/shared-kernel/pull/175
They requested for these params